andela/activo-api
# #159303925 An Endpoint To Edit A Person

#### What does this PR do?
>- Adds an endpoint to edit a person
#### Description of Task to be completed?
>- As a developer, I should be able to edit a person's details by making a request to `/centers/{<center_id>}/people/<person_id>`
#### How should this be manually tested?
>- Run `git fetch origin ft-edit-person-159303925:ft-edit-person-159303925` in your terminal, then run `git checkout ft-edit-person-159303925`
>- Run `flask run`
>- In Postman, make a PATCH request to the endpoint `localhost:5000/api/v1/centers/{center_id}/people/{person_id}` with the following payload: 
```
{
	"name": "John",
	"role": "Operations Manager"
}
```
>- You should receive the following data:
```
{
    "status": "success",
    "message": "Details for John updated successfully",
    "data": {
        "name": "John",
        "image_url": null,
        "role": "Operations Manager",
        "id": "-LIuX9JHf77Hi_XGvRSL",
        "email": "gbenga@andela.com"
    }
}
```
#### Any background context you want to provide?
>- N/A
#### What are the relevant pivotal tracker stories?
>- [#159303925](https://www.pivotaltracker.com/story/show/159303925)
#### Screenshots (if appropriate)
>- N/A
#### Postman Collection
>- [Link](https://www.getpostman.com/collections/11d3502c3f2cf3c35bd6)
#### Questions:
>-N/A
